test/integration/bad/missing_dist_suffix/stanc.expected

```diff
@@ -20,16 +20,10 @@ Semantic error in 'completely_undefined2.stan', line 5, column 12 to column 23:
 
 Function 'foo_lpdf' is not implemented for distribution 'foo', use 'foo_lpmf' instead.
   $ ../../../../../install/default/bin/stanc deprecated_suffix.stan
-Semantic error in 'deprecated_suffix.stan', line 4, column 14 to column 40:
-   -------------------------------------------------
-     2:  }
-     3:  model {
-     4:      target += von_mises_ccdf_log(1, 0,1);
-                       ^
-     5:  }
-   -------------------------------------------------
-
-Function 'von_mises_ccdf_log' is not implemented for distribution 'von_mises'.
+Warning in 'deprecated_suffix.stan', line 4, column 14: von_mises_ccdf_log is
+    deprecated and will be removed in Stan 2.32.0. Use von_mises_lccdf
+    instead. This can be automatically changed using the canonicalize flag
+    for stanc
```

Comment:
Can we update all these tests? 

The intention of this one is to use suffixes. Do we have any probability families which still don't define cdf/lcdf/lccdf functions?

Reply:
Ah, I missed these changes. Yeah, all of distrubitons listed with [Lpmf] or [Lpdf] don't have cdfs defined: https://github.com/stan-dev/stanc3/blob/9218def9e170e4d32ef0eecef8ec92b1dfbc6fdb/src/middle/Stan_math_signatures.ml#L207

Will pick one and change.

Reply:
Fixed, thanks!